sdk/sdktypes/session_state.go

```diff
@@ -18,6 +18,7 @@ type SessionStateTraits struct{}
 
 func (SessionStateTraits) Validate(m *SessionStatePB) error {
 	return errors.Join(
+		objectField[SessionStateStopped]("canceled", m.Stopped),
```

Comment:
naming inconsistency? is session stopped or cancelled? let's use the same naming?

Reply:
fixed